tests/test_proxy.py

```diff
@@ -1,6 +1,7 @@
 """Tests for the base traefik proxy"""
 
 import pytest
+from proxytest import *  # noqa
```

Comment:
```suggestion
# Some proxy tests are defined in `proxytest` so that they can be used in external repositories
from proxytest import *  # noqa
```
I've based this comment on this changelog entry, I can't find any other documented explanation:
https://github.com/jupyterhub/traefik-proxy/blob/5b06cc1ebee3233e4fc4cc4784aac1d0bf4eb994/docs/source/changelog.md?plain=1#L136

Reply:
I think this won't be true now that we've moved the tests into `tests` out of the package.

Maybe the thing to do is to merge the two files, since the way to do the other is probably via `pytest-jupyterhub` now.